lib/zhr_devs/submissions/events/solution_submitted.ex

```diff
@@ -4,12 +4,12 @@ defmodule ZhrDevs.Submissions.Events.SolutionSubmitted do
   """
 
   alias Uptight.Base.Urlsafe
-  alias Uptight.Text, as: T
+  alias Uptight.Text.Urlencoded
```

Comment:
Probably here and everywhere else can be TUrl or TU. But it may be a bit too Haskelly. I'd go with TU to be honest.